templates/clients.tmpl

```diff
@@ -40,13 +41,13 @@ func New{{ $name }}Stub() {{ $name }}Client {
 {{- if not (IsNil $method.Output) }}
 {{- if not (IsDuplicate "Output" (print $method.Output.Package "." $method.Name)) }}
 {{ $prefix := CapitalizeFirstLetter $method.Output.Package }}
-type {{ $prefix }}{{ $method.Name }}Result struct {
-	Result workflow.Future
+type {{ $prefix }}{{ $method.Name }}Future struct {
+	Future workflow.Future
```

Comment:
Is this supposed to be exported? Should consumers access the future directly?

Reply:
Added  a comment that explains that it is needed for Selector.addFuture